docs/UserGuide.adoc

```diff
@@ -73,16 +75,146 @@ Format: `help`
 
 === Pre-Trip Preparation
 
-Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Allows the user to handle transportation, accommodation bookings as well as insurance when preparing for a trip.
 
-[TIP]
-A person can have any number of tags (including 0)
+==== Transportation Bookings
+
+The following sections [3.2.1.1 to 3.2.1.3] allows the user to set and modify transportation bookings in the trip.
+
+===== Create new transportation booking: `addtransport`
+
+Creates a new transportation booking for the trip.
+
+Format: `addtransport day/DAY type/TYPE departtime/TIME_OF_DEPARTURE arrivetime/TIME_OF_ARRIVAL startlocation/START_LOCATION
+         endlocation/END_LOCATION`
+
+****
+* `TIME_OF_DEPARTURE` and `TIME_OF_ARRIVAL` needs to be in 24hr format.
+****
+
+Examples:
+
+* `addtransport day/1 type/Airplane departtime/0900 arrivetime/1500 startlocation/Singapore endlocation/Japan` +
+Creates an airplane booking for the first day that starts at 0900hr and ends at 1500hr.
+The location starting from Singapore and ending in Japan.
+
+
+===== Edit transportation booking: `edittransport`
+
+Edit a transportation booking in the trip.
+
+Format: `edittransport INDEX [day/DAY] [type/TYPE] [departtime/TIME_OF_DEPARTURE] [arrivetime/TIME_OF_ARRIVAL]
+        [startlocation/START_LOCATION] [endlocation/END_LOCATION]`
```

Comment:
For this feature, I am a bit unsure which is the best way. 

**Option 1:**
- Inlcude `day/DAY`, `departtime/TIME_OF_DEPARTURE`, `arrivetime/TIME_OF_ARRIVAL`, where time of departure and arrival are 24hr time.
Pro: Easy to implement
Con: Need multiple entries for a single booking that spans over the 00:00 time mark.

**Option 2:**
- Include `departtime/TIME_OF_DEPARTURE`, `arrivetime/TIME_OF_ARRIVAL`, where time of departure and arrival are date and time.
Pro: Easy to enter for the user, especially when the booking spans over the 00:00 time mark.
Con: Difficult to implement if we need to attach the information to the `Day` in which it spans over.

My original idea was option 2 because option 1 will result in multiple entry for one booking which seems strange. However, I think you can go with either.

Reply:
I was stuck in figuring out whether it needed the day input as well, because my other idea is to have the departure and arrival time take in a date time format